scripts/localStorageUtils.js

```diff
@@ -10,7 +10,7 @@ const getLocalStorage = (key) => {
 };
 
 const setLocalStorage = (key, data) => {
-  window.localStorage.setItem(key, JSON.stringify(data));
+  window.sessionStorage.setItem(key, JSON.stringify(data));
```

Comment:
unction name - `setLocalStorage` and usage of `sessionStorage` within function body doesn't match up